lms/services/auto_grading.py

```diff
@@ -143,7 +143,7 @@ def calculate_grade(
                 raise ValueError("Unknown auto grading configuration")  # noqa: EM101, TRY003
 
         grade = min(1, grade)  # Proportional grades are capped at 1
-        return round(grade, 2)
+        return round(grade, 3)
```

Comment:
This is the only change on this PR.

We'll round to three decimal places instead of 2 for the resulting grade of auto grading.

Reply:
The issue is documented well in this PR, but linking to the issue in a comment would also be useful, to point out to future readers that the precision is deliberate.